lib/elixir/lib/path.ex

```diff
@@ -207,7 +207,7 @@ defmodule Path do
 
       Path.type("D:/usr/local/bin") #=> :absolute
       Path.type("usr/local/bin")    #=> :relative
-      Path.type("D:bar.ex")         #=> :volumerelative
+      Path.type("D:/bar.ex")         #=> :volumerelative
```

Comment:
```suggestion
      Path.type("D:/bar.ex")        #=> :volumerelative
```

Reply:
Good eyes.

Reply:
`D:bar.ex` is actually valid on Windows. More info here: https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats